plugins/vli/fu-vli-usbhub-device.c

```diff
@@ -623,6 +623,19 @@ fu_vli_usbhub_device_guess_kind(FuVliUsbhubDevice *self, GError **error)
 	return TRUE;
 }
 
+static gboolean
+_fu_usb_device_has_child(FuUsbDevice *self, guint16 vid, guint16 pid)
```

Comment:
shouldn't this be in `fu-usb-device.c` and a new symbol?

Reply:
Not sure if it's general enough API; I wanted to see if the idea worked first :)

Reply:
It looks like `g_usb_device_get_children` is first time being used, yes it worked well with `libgusb` 0.3.10.  With this function being used a segfault is seen with 0.3.5, I didn't spend time to find out what the minimum compatible version is. We need uprev the dependency to prevent this bug, see below when `g_usb_device_get_children` get called.

```
15:45:18:0327 FuDevice             no registered custom flag usb3 on FuUsbDevice
15:45:18:0327 FuDevice             no registered custom flag has-shared-spi-pd on FuUsbDevice
15:45:18:0329 FuDevice             already set GType to FuVliUsbhubDevice, ignoring FuVliUsbhubDevice
15:45:18:0338 FuDevice             no registered custom flag usb3 on FuUsbDevice
15:45:18:0339 FuDevice             no registered custom flag has-shared-spi-pd on FuUsbDevice
15:45:18:0340 FuDevice             already set GType to FuVliUsbhubDevice, ignoring FuVliUsbhubDevice
15:45:18:0350 FuDevice             no registered custom flag usb3 on FuUsbDevice
15:45:18:0350 FuDevice             no registered custom flag has-shared-spi-pd on FuUsbDevice
15:45:18:0352 FuDevice             already set GType to FuVliUsbhubDevice, ignoring FuVliUsbhubDevice
15:45:18:0361 FuDevice             no registered custom flag usb3 on FuUsbDevice
15:45:18:0361 FuDevice             no registered custom flag has-shared-spi-pd on FuUsbDevice
15:45:18:0363 FuDevice             already set GType to FuVliUsbhubDevice, ignoring FuVliUsbhubDevice
...
...
```